src/server/journal/serializer.cc

```diff
@@ -75,6 +75,7 @@ void JournalWriter::Write(const journal::Entry& entry) {
     case journal::Op::SELECT:
       return Write(entry.dbid);
     case journal::Op::COMMAND:
+    case journal::Op::EXPIRED:
```

Comment:
Check the snapshoting flow. Currently we only write COMMAND and MULTI_COMMAND to journal inside SliceSnapshot::OnJournalEntry

Reply:
what about this comment?

Reply:
Indeed, fixed it